lib/matplotlib/colors.py

```diff
@@ -1190,6 +1190,13 @@ class ListedColormap(Colormap):
 
         the list will be extended by repetition.
     """
+
+    @_api.delete_parameter(
+        "3.11", "N",
+        message="Passing 'N' to ListedColormap is deprecated since %(since)s "
+                "and will be removed in %(removal)s. Please process the list "
+                "of passed colors yourself if needed."
```

Comment:
```suggestion
                "and will be removed in %(removal)s. Please ensure the list "
                "of passed colors is the required length instead."
```
To be more explicit.